src/components/Header/HeaderUserMenu/HeaderUserMenu.test.js

```diff
@@ -74,6 +74,32 @@ describe('Header user menu', () => {
     expect(getByText('upgrade')).toBeInTheDocument();
   });
 
+  it('user has a prepaid plan', () => {
```

Comment:
This should be a test reinforcing that when the plan has a button option de button must be displayed. It goes beyond prepaid plan. 
```suggestion
  it('user has a button action it must be displayed', () => {
```

Reply:
```suggestion
  it('should render a button if the plan has buttonText and buttonUrl properties', () => {
```

Reply:
The last one I think it's best.